test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
```

Comment:
since we have a possibility to add multiple commands inside one plugin, I'd compose a new PR to replace src/action by src/link